### DISCUSSION TITLE
How to check if a lua/api minimal required version?

### DISCUSSION THREAD

**Discussion by linrongbin16** [neovim/neovim#24488] [Q&A] [answered]
Hi, since Neovim is rapidly growing, and lua/api is changing.

For example I want to write a plugin that is using these lua methods:
* `vim.inspect`
* `vim.api.nvim_get_commands`

But how should I check the minimal Neovim version that support these two APIs? (thus I can declare the minimal required version)

**Comment by zeertzjq** [comment, marked as answer]
You may be interested in <https://github.com/tweekmonster/helpful.vim>, although it's based on help tags.

**Reply by linrongbin16** [reply]
Thank you!
